packages/manager/modules/product-offers/src/workflows/product-offers-order-workflow.class.js

```diff
@@ -174,13 +176,17 @@ export default class OrderWorkflow extends Workflow {
         isString(serviceName) && !isEmpty(serviceName)
           ? this.WucOrderCartService.addProductServiceOptionToCart(
               this.cartId,
-              this.productName,
+              isFunction(this.productName)
+                ? this.productName()
+                : this.productName,
```

Comment:
**nitpick:** This is pretty confusing. Can we just maintain `this.productName` to be always string.

Reply:
In fact i need to have something to dynamic. Because in my order funnel database the all type of database haven't same product name and one need service name because is an option other no because is individual service.